python/cudf_polars/cudf_polars/experimental/select.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+"""Parallel Select Logic."""
+
+from __future__ import annotations
+
+import operator
+from functools import reduce
+from typing import TYPE_CHECKING
+
+from cudf_polars.dsl.ir import Select
+from cudf_polars.dsl.traversal import traversal
+from cudf_polars.experimental.base import PartitionInfo
+from cudf_polars.experimental.dispatch import lower_ir_node
+
+if TYPE_CHECKING:
+    from collections.abc import MutableMapping
+
+    from cudf_polars.dsl.ir import IR
+    from cudf_polars.experimental.parallel import LowerIRTransformer
+
+
+_PARTWISE = (
+    "Literal",
+    "LiteralColumn",
+    "Col",
+    "ColRef",
+    "BooleanFunction",
+    "StringFunction",
+    "TemporalFunction",
+    "Filter",
+    "Cast",
+    "Ternary",
+    "BinOp",
+    "UnaryFunction",
+)
```

Comment:
I don't actually know the specific `Expr` sub-types that are "partition-wise" - This was my initial "guess".

Reply:
OK, I should open a PR that classifies things appropriately.